### DISCUSSION TITLE
MAX31865 MQTT

### DISCUSSION THREAD

**Discussion by ChristosKeleshis** [arendst/Tasmota#15501] [Q&A] [answered]
Hello,

I have a MAX31865 connected to a Wemos D1 running Tasmota. From the Console i receive the following tele every 5 minutes as expected:
MQT: tele/BoilerTemperature/SENSOR = {"Time":"2022-05-01T14:12:02","MAX31865-0":{"Temperature":28.3,"Resistance":111.0,"Error":0},"TempUnit":"C"}

When i ask for one of the parameters i can only get the time as follows
14:16:16.338 CMD: stat/BoilerTemperature/SENSOR/Time
14:16:16.344 MQT: stat/BoilerTemperature/RESULT = {"Time":"2022-05-01T14:16:16"}

The problem is that when i try to get Temperature, or Temperature unit i get the following errors as follows:
14:18:48.176 CMD: stat/BoilerTemperature/SENSOR/MAX31865-0/Temperature
14:18:48.184 MQT: stat/BoilerTemperature/RESULT = {"Command":"Unknown"}
14:19:03.824 CMD: stat/BoilerTemperature/SENSOR/TempUnit
14:19:03.832 MQT: stat/BoilerTemperature/RESULT = {"Command":"Unknown"}

What is wrong with these commands?

Thanks,

Christos

**Comment by sfromis** [comment, marked as answer]
You cannot "ask" for any of the parameters in the JSON payload. While `Time` is a valid command, that has nothing to do with what you have in the JSON payload. As the values are already published to MQTT, there is no need for "asking", but you can use `TelePeriod` to vary how often they're published, and you can issue the command `Status 10` to have extra sensor payloads published. You can also create Tasmota rules to get sensor values, and react to them.

If you want to execute commands, you should **not** make something like a topic string. Like when the actual command is just `Time` , not `stat/BoilerTemperature/SENSOR/Time`. Due to a quirk of tasmota, whatever "nonsense" you may have before the last slash is just ignored.

**Reply by ascillato** [reply]
> What is wrong with these commands?

Those are not Tasmota commands. Please, check the Tasmota documentation for valid commands.
